client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -40,18 +43,22 @@ class PluginAutomatedTransfer extends Component {
 	};
 
 	componentWillMount() {
-		const { COMPLETE, CONFLICTS } = transferStates;
-		const { isTransferring, transferState } = this.props;
+		const { COMPLETE } = transferStates;
+		const {
+			isTransferring,
+			isTransferFailed,
+			transferState,
+		} = this.props;
 
 		if ( COMPLETE === transferState ) {
 			this.setState( { transferComplete: true } );
-		} else if ( isTransferring || CONFLICTS === transferState ) {
+		} else if ( isTransferring || isTransferFailed ) {
```

Comment:
not sure. probably better overall to rename the others. I think I may have named them myself and was thinking, "is this a failed automated transfer" instead of "did this automated transfer fail."

_if_ we focus on the transfer it should probably be `isFailedTransfer` but if on the action then probably `didTransferFail` or something like that.

let's just keep the changes minimal for now and let the rest stew in our minds. no need to grow the PR over naming.